client/web/web.go

```diff
@@ -280,6 +293,15 @@ func Handle(w http.ResponseWriter, r *http.Request) {
 		io.WriteString(w, authenticationRedirectHTML)
 		return
 	}
+	switch {
+	case strings.HasPrefix(r.URL.Path, "/localapi/"):
+		if c.APIHander == nil {
```

Comment:
if we remove the apihandler type, then even a zero value tailscale.LocalClient should function, so I don't think we'd need any special error handling here?